src/middleware/events.coffee

```diff
@@ -3,7 +3,12 @@ logger = require 'winston'
 events = require '../model/events'
 messageStore = require '../middleware/messageStore'
 config = require "../config/config"
-config.visualizer = config.get('visualizer')
+config.events = config.get('events')
+
+if !config.events
+  # maybe we're using outdated config
+  config.events = config.get('visualizer')
```

Comment:
Oh, what I was trying to do here was backwardly supporting the older config format: `if !newConfig then tryReadOldFormat`. I guess if we're removing the old visualizer API, then there's no reason to do this though. What do you think? Keeping it will make upgrading easier for people though?